tests/model/test_rf_model.py

```
@@ -1,80 +1,89 @@
 import numpy as np
+import pandas as pd
 import pytest
 from sklearn.compose import ColumnTransformer
 from sklearn.preprocessing import StandardScaler
 from sklearn.pipeline import Pipeline
 
 from power_consumption.model import ConsumptionModel
-from power_consumption.config import Config, Hyperparameters, Features, Target, Dataset
+from power_consumption.config import Config
+from power_consumption.preprocessing.data_preprocessor import DataProcessor
+from loguru import logger
```

Ruff

`loguru.logger` imported but unused

Remove unused import: `loguru.logger`

(F401)